### PR TITLE
LOOFAH-30 add Grade object and queries

### PR DESCRIPTION
**Pull request by vijamesDD**

### CODE REVIEW

File: src/main/resources/skills.graphqls [hunk 1]
```diff
@@ -10,6 +10,8 @@ type Query {
   skillsByCategory(categoryId: String): [Skill!]!
   craft(id: String): Craft!
   crafts: [Craft!]!
+  grade(id: String): Grade
```
Comment (elishkaflint) [inline src/main/resources/skills.graphqls]: should it be `Grade!`?

File: src/test/resources/testQueries/gradeQuery.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+query {
```
Comment (elishkaflint) [inline src/test/resources/testQueries/gradeQuery.txt:1]: I might be missing it but is there an integration test that uses this test query?

File: src/test/resources/testQueries/gradesQuery.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+query {
```
Comment (elishkaflint) [inline src/test/resources/testQueries/gradesQuery.txt:1]: I might be missing it but is there an integration test that uses this test query?

**Comment by vijamesDD** [comment]
Added integration tests.

**Review by elishkaflint — APPROVED** [review]

File: src/test/resources/testQueries/gradesQuery.txt (hunk 3 above)
Comment (vijamesDD) [reply src/test/resources/testQueries/gradesQuery.txt:1]: yep I've put `returns_all_seeded_grades()` in `src/test/java/com/loofah/graph/api/integrationTests/IntegrationTest.java` which uses this

File: src/test/resources/testQueries/gradeQuery.txt (hunk 2 above)
Comment (vijamesDD) [reply src/test/resources/testQueries/gradeQuery.txt:1]: Yep I've put `returns_correct_grade_when_id_valid()` in `src/test/java/com/loofah/graph/api/integrationTests/IntegrationTest.java`

File: src/main/resources/skills.graphqls (hunk 1 above)
Comment (vijamesDD) [reply src/main/resources/skills.graphqls]: Good spot, yes I think so. Have updated

**Pull request merged**